content/instructor/style-guide/accessible-speach-with-explicit-references/index.mdx

```diff
@@ -0,0 +1,28 @@
+---
+date: 2019-11-27
+title: 'Accessible Speech'
+description: 'Learn how to use explicit references in your speech for clearer teaching'
+categories: ['instructor', 'style']
+published: true
+shareImage: 'https://res.cloudinary.com/dg3gyk0gu/image/upload/v1571260698/og-image-assets/share_image_style.png'
+redirect_from:
+  - /accessible-speech/
```

Comment:
Also copied this from another file, does the root level get created for every article?

Reply:
This is only needed if it was a previously published article that you moved.

Reply:
This is only needed if it was a previously published article that you moved.